test/setup.spec.js

```diff
@@ -355,4 +355,46 @@ describe('setup', () => {
       })
       .then(done);
   });
+
+  describe('Bindings', ()=>{
+    it('should bind the vm when calling with parentesis', async ()=>{
+      let context = null;
+      const contextFunction = jest.fn(function (){
+        context = this
+      });
+
+      const vm = new Vue({
+        template: '<div><button @click="contextFunction()"/></div>',
+        setup() {
+          return {
+            contextFunction
+          }
+        }
+      }).$mount();
+  
+      await vm.$el.querySelector('button').click();
+      expect(contextFunction).toBeCalled();
+      expect(context.$el).toBe(vm.$el);
```

Comment:
this could be moved inside of the `contextFunction` and get rid of the context variable

Reply:
also, shouldn't we just compare to `vm` directly?

Reply:
when I created the tests `vm` was being proxied, but now because of the `bind` I can do that 👍